kolibri/plugins/learn/assets/src/views/card-grid/grid-item/index.vue

```diff
@@ -112,6 +112,7 @@
   $card-text-padding = ($card-width / (320 / 24))
   $card-elevation-resting = 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 3px 1px -2px rgba(0, 0, 0, 0.2), 0 1px 5px 0 rgba(0, 0, 0, 0.12)
   $card-elevation-raised = 0 8px 10px 1px rgba(0, 0, 0, 0.14), 0 3px 14px 2px rgba(0, 0, 0, 0.12), 0 5px 5px -3px rgba(0, 0, 0, 0.2)
+  $elevation-transition = all 0.4s cubic-bezier(0.25, 0.8, 0.25, 1)
```

Comment:
When possible, it's best to avoid `all` as a transition target. Better to be explicit about the intended property